build/targets.simulator.ios.mak

```diff
@@ -0,0 +1,10 @@
+ifndef ARCH
+HANDY_TARGETS_EXTENSIONS += apk
```

Comment:
ipa?

Reply:
Well, not even that. Those are in `build/platform.*.mak` now 😄